components/builder-api/src/server/services/github.rs

```diff
@@ -45,7 +46,7 @@ pub enum GitHubEvent {
 impl FromStr for GitHubEvent {
     type Err = Error;
 
-    fn from_str(event: &str) -> Result<Self, Self::Err> {
+    fn from_str(event: &str) -> error::Result<Self, Error> {
```

Comment:
I think this can just be `fn from_str(event: &str) -> Result<Self, Error>` since we are importing the result from the error.rs

Reply:
No, it's specifically not our Result.. it takes 2 arguments..